<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Citra">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Proses">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_loadCitra">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Citra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prosesCitra">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Proses Citra</string>
    </property>
   </widget>
   <widget class="QSlider" name="brightnessSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelbrightness">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buutonBrightness">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Brighness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buutonKontras">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kontras</string>
    </property>
   </widget>
   <widget class="QSlider" name="kontrasSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelkontras">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="action_operasiPencerahan"/>
    <addaction name="action_aturKontras"/>
    <addaction name="action_regangkanKontras"/>
    <addaction name="action_negativeImage"/>
    <addaction name="action_binerImage"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="action_histogramGrayscale"/>
    <addaction name="action_histogramRGB"/>
    <addaction name="action_histogramEqualization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuRotasi">
     <property name="title">
      <string>Rotasi</string>
     </property>
     <addaction name="action90_Derajat"/>
     <addaction name="action_90_Derajat"/>
     <addaction name="action45_Derajat"/>
     <addaction name="action_45_Derajat"/>
     <addaction name="action180_Derajat"/>
    </widget>
    <widget class="QMenu" name="menuResize">
     <property name="title">
      <string>Resize</string>
     </property>
     <addaction name="actionZoom_in"/>
     <addaction name="actionZoom_out"/>
    </widget>
    <addaction name="action_translasi"/>
    <addaction name="menuRotasi"/>
    <addaction name="menuResize"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuAritmatika">
    <property name="title">
     <string>Aritmatika</string>
    </property>
    <addaction name="actionTambah_Kurang_Kali_Bagi"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Boolean">
    <property name="title">
     <string>Operasi Boolean</string>
    </property>
    <addaction name="actionAnd"/>
    <addaction name="actionOr"/>
    <addaction name="actionXOR"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuHistogram"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuAritmatika"/>
   <addaction name="menuOperasi_Boolean"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_operasiPencerahan">
   <property name="text">
    <string>Kecerahan</string>
   </property>
  </action>
  <action name="action_aturKontras">
   <property name="text">
    <string>Tambah Kontras</string>
   </property>
  </action>
  <action name="action_regangkanKontras">
   <property name="text">
    <string>Regangkan Kontras</string>
   </property>
  </action>
  <action name="action_negativeImage">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="action_binerImage">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_histogramGrayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="action_histogramRGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="action_histogramEqualization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="action_translasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="action90_Derajat">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action_90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action45_Derajat">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action_45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTambah_Kurang_Kali_Bagi">
   <property name="text">
    <string>Tambah Kurang Kali Bagi</string>
   </property>
  </action>
  <action name="actionAnd">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionOr">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
